src/test/java/PlayingCardUnitTests.java

```diff
@@ -0,0 +1,49 @@
+import PlayingCard.Card;
+import PlayingCard.CardSuit;
+import PlayingCard.CardValue;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlayingCardUnitTests {
+
+    @Test
+    @DisplayName("Check if multiple of the same card is found in deck")
+    public void multipleInDeckCheck() {
+        var testDeck = Main.createDeck();
```

Comment:
Strictly speaking you're testing Main here, whereas we're only interested in Card for now